src/vendor/azure-uamqp-c/src/message_sender.c

```diff
@@ -442,6 +443,62 @@ static SEND_ONE_MESSAGE_RESULT send_one_message(MESSAGE_SENDER_INSTANCE* message
                 }
                 break;
             }
+
+            case MESSAGE_BODY_TYPE_SEQUENCE:
```

Comment:
It might be a good idea to run some cross language tests to ensure that the "sequence body" messages that we're generating here are recognized and consumable by a different AMQP implementation.

Reply:
(And in reverse)

Reply:
agreed, we'll do a cross-language test